guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -0,0 +1,20 @@
+Adam Růžička
```

Comment:
For Foreman we don't have a line for each contributor, but rather combine them.  I'm not sure which is best, but I'd prefer them to be consistent.